src/Schema/Table.php

```diff
@@ -704,7 +622,23 @@ public function getPrimaryKeyColumns()
             throw new DBALException('Table ' . $this->getName() . ' has no primary key.');
         }
 
-        return $primaryKey->getColumns();
+        return $this->filterColumns($primaryKey->getColumns());
+    }
+
+    /**
+     * Returns the foreign key columns
+     *
+     * @return Column[]
+     */
+    public function getForeignKeyColumns()
```

Comment:
Should it declare the return type?

Reply:
This was a moved method, it'll break BC by doing it if anyone extends Table.